test-harness/src/test/java/com/netflix/conductor/tests/utils/SqlServerTestRunner.java

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
This file doesn't appear to be used. In other modules it's used in integration tests (end to end). Perhaps we add an end to end integration test?

Reply:
There are now integration tests for sqlserver
https://github.com/TwoUnderscorez/conductor/commit/44bcafaac894648992c2e16e754a252d35d8e79b